### PR TITLE
Closes AgileVentures/MetPlus_tracker#424

### PR DESCRIPTION
**Pull request by patmbolger**
As a company person
I want to see my open and completed tasks

This story includes these tasks:
- [x] Convert to the tabbed format.  For company admin there would be 4 tabs (ala agency admin) and for company contact there would be two tabs (ala job developer).
- [x] Allow a company admin to assign tasks to other company people (e.g., review job application) 
- [x] Confirm that tasks show up in the appropriate tabs as they undergo state changes, for both company admin and (assigned) company contact.
- [x] Add styling to tab structure to make it standout and distinct from surrounding information.
- [x] Add / edit acceptance tests

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -14,10 +14,31 @@
  *= require_self
  */
 
- /*homepage css*/
-.pagination{
```
Comment (joaopapereira) [inline app/assets/stylesheets/application.css]: We no longer use this?

File: app/models/company.rb [hunk 2]
```diff
@@ -54,6 +54,10 @@ def self.company_admins(company)
     find_users_with_role(company, CompanyRole::ROLE[:CA])
   end
 
+  def self.everyone(company)
```
Comment (joaopapereira) [inline app/models/company.rb]: do we have tests for this function?

File: app/models/concerns/task_manager/business_logic.rb [hunk 3]
```diff
@@ -91,6 +91,8 @@ def assignable_list
       info = ASSIGNABLE_LIST[task_type.to_sym]
       if info[:type] == :agency
         return Agency.send info[:function], task_owner[0].agency
+      elsif info[:type] == :company
```
Comment (joaopapereira) [inline app/models/concerns/task_manager/business_logic.rb]: Do we have tests for this change?

File: app/models/task.rb [hunk 4]
```diff
@@ -87,6 +89,22 @@ def self.find_by_company_open user
     end
   end
 
+  def self.find_by_company_new user
+    new_tasks.company_tasks(user)
```
Comment (joaopapereira) [inline app/models/task.rb]: Do we have tests for this function?

File: app/models/task.rb [hunk 5]
```diff
@@ -87,6 +89,22 @@ def self.find_by_company_open user
...
+  end
+
+  def self.find_by_company_active user
```
Comment (joaopapereira) [inline app/models/task.rb]: Do we have tests for this function?

File: app/models/task.rb [hunk 6]
```diff
@@ -87,6 +89,22 @@ def self.find_by_company_open user
...
+    active_tasks.company_tasks(user)
+  end
+
+  def self.find_by_company_closed user
```
Comment (joaopapereira) [inline app/models/task.rb]: Do we have tests for this function?

File: app/views/company_people/home.html.haml [hunk 7]
```diff
@@ -24,9 +24,44 @@
                      admin_aa: @admin_aa,
                      company_admins: @company_admins}
 
+-# .col-sm-12
+-#   %h3 Tasks that need your attention
```
Comment (joaopapereira) [inline app/views/company_people/home.html.haml]: Do we need to keep this commented code?

File: features/company_person.feature [hunk 8]
```diff
@@ -172,3 +177,18 @@ Feature: Company Person
...
+    And I should see "Your Open Tasks"
+    And I should see "Review job application"
+    And the task 1 status is "Assigned"
+    Then I press the wip button of the task 1
+    And I wait 5 seconds
+    And I should see notification "Work on the task started"
+    And the task 1 status is "Work in progress"
+    Then I press the done button of the task 1
+    And I wait 1 second
+    And I should see notification "Work on the task is done"
```
Comment (joaopapereira) [inline features/company_person.feature]: Should we ensure that the tast is no longer visible?

File: app/models/concerns/task_manager/business_logic.rb (hunk 3 above)
Comment (patmbolger) [inline app/models/concerns/task_manager/business_logic.rb]: Added tests to `concerns/task_manager/business_logic_spec.rb` for `assignable_list`.

File: app/models/task.rb (hunk 4 above)
Comment (patmbolger) [inline app/models/task.rb]: Added tests to task_spec.rb.

File: app/models/task.rb (hunk 6 above)
Comment (patmbolger) [inline app/models/task.rb]: Added tests to task_spec.rb.

File: features/company_person.feature (hunk 8 above)
Comment (patmbolger) [inline features/company_person.feature]: added a step

**Comment by joaopapereira** [comment]
Going to merge

**Pull request merged**